pkg/reportk8s/reportk8s.go

```diff
@@ -0,0 +1,116 @@
+package reportk8s
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/aquasecurity/trivy/pkg/types"
+	"github.com/olekukonko/tablewriter"
+)
+
+type KubernetesReport struct {
```

Comment:
report should be done on trivy CLI as it not the lib concern

Reply:
yes, I agree, my thinking was that for a first version, we have more flexibility having the logic here. But happy to already start it on trivy.